### PR TITLE
Dockerfile4.9

### PR DESCRIPTION
**Pull request by pdion891**
- Fixed docker build for simulator, marvin and management-server.
- Fixed change of release script to update dockerfiles.
- I will update Jenkins builds once merged.

### CODE REVIEW

**Comment by rhtyd** [comment]
@pdion891 rebase against latest master and squash commits into a single commit, thanks

LGTM

tag:easypr

File: tools/docker/Dockerfile.centos6 [hunk 1]
```diff
@@ -18,14 +18,16 @@
 FROM centos:6
 
 MAINTAINER "Apache CloudStack" <dev@cloudstack.apache.org>
-LABEL Vendor="Apache.org" License="ApacheV2" Version="4.6.0"
+LABEL Vendor="Apache.org" License="ApacheV2" Version="4.9.0-SNAPSHOT"
 
-ENV PKG_URL=http://jenkins.buildacloud.org/job/package-rhel63-master/lastSuccessfulBuild/artifact/dist/rpmbuild/RPMS/x86_64
+ENV PKG_URL=http://74.121.245.247/job/package-master-rhel63/lastSuccessfulBuild/artifact/dist/rpmbuild/RPMS/x86_64
```
Comment (pdion891) [inline tools/docker/Dockerfile.centos6]: @rhtyd, @swill  I know this is wrong, but the new URL is not live yet:  builds.cloudstack.org so either we merge this and I'll update it at some point or we just cancel this PR until the URL go live ?

Comment (rhtyd) [inline tools/docker/Dockerfile.centos6]: I'm fine with either ways, is the jenkins server on this IP public and can we have accounts on it?

Comment (pdion891) [inline tools/docker/Dockerfile.centos6]: @rhtyd I've create you an account on the Jenkins server. not a lot of jobs are there yet. 
Lets continue the discussion on slack...

**Comment by bvbharatk** [comment]
### ACS CI BVT Run
 **Sumarry:**
 Build Number 146
 Hypervisor xenserver
 NetworkType Advanced
 Passed=68
 Failed=5
 Skipped=3

_Link to logs Folder (search by build_no):_ https://www.dropbox.com/sh/yj3wnzbceo9uef2/AAB6u-Iap-xztdm6jHX9SjPja?dl=0


**Failed tests:**
* test_vpc_vpn.py

 * ContextSuite context=TestRVPCSite2SiteVpn>:setup Failing since 11 runs

 * ContextSuite context=TestVpcRemoteAccessVpn>:setup Failing since 11 runs

 * ContextSuite context=TestVpcSite2SiteVpn>:setup Failing since 11 runs

* test_volumes.py

 * test_06_download_detached_volume Failed

* test_vm_life_cycle.py

 * test_10_attachAndDetach_iso Failed


**Skipped tests:**
test_vm_nic_adapter_vmxnet3
test_static_role_account_acls
test_deploy_vgpu_enabled_vm

**Passed test suits:**
test_deploy_vm_with_userdata.py
test_affinity_groups_projects.py
test_portable_publicip.py
test_over_provisioning.py
test_global_settings.py
test_scale_vm.py
test_service_offerings.py
test_routers_iptables_default_policy.py
test_routers.py
test_reset_vm_on_reboot.py
test_snapshots.py
test_deploy_vms_with_varied_deploymentplanners.py
test_login.py
test_list_ids_parameter.py
test_public_ip_range.py
test_multipleips_per_nic.py
test_regions.py
test_affinity_groups.py
test_network_acl.py
test_pvlan.py
test_nic.py
test_deploy_vm_root_resize.py
test_resource_detail.py
test_secondary_storage.py
test_disk_offerings.py

**Comment by pdion891** [comment]
@swill, @runseb does one of you could LGTM or comment  thanks,

**Comment by runseb** [comment]
LGTM

I still think we need to put the db in another container, and start the whole thing with compose.
Also we need to make the image smaller, maybe using Debian as base would be smaller than then full ubuntu:14.04

**Comment by pdion891** [comment]
@runseb I agree, images must be smaller. for the simulator, it's kind of usefull to have everything in it but it's not container best practice for sure.  other images, we could probably use alpine linux, but still need to figure out how to run cloudstack as a single .war file.

**Comment by pdion891** [comment]
@resmo I've updated the PR. I've tested the simulator and marvin.

**Comment by pdion891** [comment]
Hi @jburwell, do you think we can merge this in master or you are in a code freeze for 4.10.0.0 ?

Thanks,

**Comment by jburwell** [comment]
@pdion891 if possible, I would like to get this merged for 4.9.1.0.  How does this PR relate to #1789?  Also, could you please investigate the Jenkins failure and create JIRA ticket for this issue?

**Comment by jburwell** [comment]
@PaulAngus @rhtyd what are your thoughts on automating the testing of the Docker container?

**Comment by jburwell** [comment]
@pdion891 can you please investigate the Jenkins failure?

**Comment by rhtyd** [comment]
@jburwell good to have this, not sure how many people maybe using container/docker based mgmt servers?

**Comment by jburwell** [comment]
@rhtyd agreed.  I apologize for being unclear -- I would like to get this PR into 4.9.1.0, but it is not a release blocker.

**Comment by resmo** [comment]
I used it for development and testing.

**Comment by pdion891** [comment]
I've retested the management server, it work with Jenkins as the build is back online.

@jburwell, do you know if there is a way to squash a commit within the same PR ?

**Comment by jburwell** [comment]
@pdion891 this [blog](https://ariejan.net/2011/07/05/git-squash-your-latests-commits-into-one/) is  good description of squash process.

**Comment by pdion891** [comment]
This PR is not valid anymore replaced by Squashed: #1815

**Pull request closed without merging**